docs/loading.md

```diff
@@ -1,128 +1,151 @@
-# Navigation and loading
+# Navigation and Loading
 
-Playwright logically splits the process of showing a new document in the page into **navigation** and **loading**.
+Playwright can navigate to URLs and handle navigations caused by page actions. This guide covers common scenarios with navigations and page load state.
 
 <!-- GEN:toc-top-level -->
-- [Navigation](#navigation)
-- [Loading](#loading)
-- [Common scenarios](#common-scenarios)
-- [Loading a popup](#loading-a-popup)
+- [Lifecycle](#lifecycle)
+- [Navigating to URL](#navigating-to-url)
+- [Navigation by page click](#navigation-by-page-click)
 <!-- GEN:stop -->
 
-## Navigation
+## Lifecycle
```

Comment:
How about "Navigation vs. Loading"?

Reply:
Thanks for the review! Reworking some bits.